cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -586,30 +588,32 @@ parameter_types! {
 			epoch: 0,
 		},
 	};
-	pub const MaxExecutionHeadersToKeep:u32 = 1000;
 }
 
-#[cfg(feature = "beacon-spec-mainnet")]
+#[cfg(not(feature = "fast-runtime"))]
 parameter_types! {
 	pub const ChainForkVersions: ForkVersions = ForkVersions {
 		genesis: Fork {
-			version: [0, 0, 16, 32], // 0x00001020
+			version: [144, 0, 0, 111], // 0x90000069
```

Comment:
Oh no, I missed updating the fork versions for Sepolia. 😞 Thanks for catching this.

Reply:
Yeah, we need this fix for the launch.